libraries/stdlib/jvm/src/kotlin/system/Timing.kt

```diff
@@ -17,9 +17,7 @@ public inline fun measureTimeMillis(block: () -> Unit): Long {
     contract {
         callsInPlace(block, InvocationKind.EXACTLY_ONCE)
     }
-    val start = System.currentTimeMillis()
-    block()
-    return System.currentTimeMillis() - start
+    return measureNanoTime(block) / 1_000_000L
```

Comment:
I'm debating if `convertDurationUnit` should be used instead, though this is likely quicker.